teptour.js

```diff
@@ -540,27 +540,27 @@ instead_of(({verb, dir}) => (verb === "looking toward" && dir === "down"
 
 world.no_go_msg.add_method({
   when: (x, dir) => x === "253 Commonwealth Ave",
-  handle: (x, dir) => `You head away from tEp, but you begin to realize the
+  handle: (x, dir) => `You head away from tEp/Xi, but you begin to realize the
 rest of the neighborhood is just a dense bank of fog. You've heard this
-virtual tEp is hosted in the cloud, and if you go that way you might fall off!`
+virtual tEp/Xi is hosted in the cloud, and if you go that way you might fall off!`
 });
 
-def_obj("tEp", "backdrop", {
-  proper_named: "tEp",
+def_obj("tEp/Xi", "backdrop", {
```

Comment:
It's not really necessary to update the object name, since it's only used as an internal key to link things up.

No harm in changing it though.